### PR TITLE
clarify remove encryption for different OME ver

### PR DESCRIPTION
**Pull request by samschan-ms**
@KCCross  please review this

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/define-mail-flow-rules-to-encrypt-email.md [hunk 1]
```diff
@@ -89,11 +89,11 @@ You can define mail flow rules for triggering message encryption with the new OM
 
 8. Choose **Save**.
 
-## Create mail flow rules to remove encryption for outgoing email messages with the new OME capabilities
+## Create mail flow rules to remove encryption for email messages with the new OME capabilities
 
-You can define mail flow rules for triggering remove message encryption with the new OME capabilities by using the EAC.
+You can define mail flow rules for triggering remove message encryption with the new OME capabilities by using the EAC or Windows PowerShell cmdlets.  When your email users send encrypted messages, you can define these rules to decrypt messages that are from within your organization.  You cannot decrypt encrypted messages originating from outside of your organization.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/define-mail-flow-rules-to-encrypt-email.md]:
```suggestion
You can define mail flow rules for triggering remove message encryption with the new OME capabilities by using the EAC or Windows PowerShell cmdlets.  When your email users send encrypted messages, you can define these rules to decrypt messages that are from within your organization.  You cannot decrypt encrypted messages originating from outside your organization.
```

File: microsoft-365/compliance/define-mail-flow-rules-to-encrypt-email.md [hunk 2]
```diff
@@ -169,12 +169,14 @@ If you haven't yet moved your organization to the new OME capabilities, use thes
...
 
-### Remove encryption from email replies encrypted without the new OME capabilities
+### Remove encryption from email without the new OME capabilities
 
-When your email users send encrypted messages, recipients of those messages can respond with encrypted replies. You can create mail flow rules to automatically remove encryption from replies so email users in your organization don't have to sign in to the encryption portal to view them. You can use the EAC or Windows PowerShell cmdlets to define these rules. If you are not yet using the new OME capabilities, you can only decrypt messages that are either sent from within your organization or messages that are replies to messages sent from within your organization. You cannot decrypt encrypted messages originating from outside of your organization.
+You can define mail flow rules for triggering remove message encryption without the new OME capabilities by using the EAC or Windows PowerShell cmdlets.  When your email users send encrypted messages, you can define these rules to decrypt messages from within your organization. You cannot decrypt encrypted messages originating from outside of your organization.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/define-mail-flow-rules-to-encrypt-email.md]:
```suggestion
You can define mail flow rules for triggering remove message encryption without the new OME capabilities by using the EAC or Windows PowerShell cmdlets.  When your email users send encrypted messages, you can define these rules to decrypt messages from within your organization. You cannot decrypt encrypted messages originating from outside your organization.
```

File: microsoft-365/compliance/define-mail-flow-rules-to-encrypt-email.md [hunk 3]
```diff
@@ -169,12 +169,14 @@ If you haven't yet moved your organization to the new OME capabilities, use thes
...
 
 #### Use the EAC to create a rule for removing encryption from email replies encrypted without the new OME capabilities
 
+When your email users send encrypted messages without the new OME capabilities, recipients of those messages need to sign in to the encryption portal to view and respond the encrypted replies. You can create mail flow rules to automatically remove encryption from replies so email users in your organization can view the replies directly in the mail client.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/define-mail-flow-rules-to-encrypt-email.md]:
```suggestion
When your email users send encrypted messages without the new OME capabilities, recipients of those messages need to sign in to the encryption portal to view and respond to the encrypted replies. You can create mail flow rules to automatically remove encryption from replies so email users in your organization can view the replies directly in the mail client.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@KCCross Please merge this PR. Thanks.

**Comment by KCCross** [comment]
Samson, we need to talk about this. I'm pretty sure we covered this elsewhere in several other ways.

**Pull request closed without merging**